modules/EnsEMBL/Web/SpeciesDefs.pm

```diff
@@ -538,11 +538,22 @@ sub _read_species_list_file {
   return $spp_list;
 }
 
+sub _get_cow_defaults {
+## Copy-on-write hash (only used by NV)
+## Note: use method instead of an 'our' variable, as the latter can be a pain
+## when shared across plugins
+  return {};
+}
+
 sub _read_in_ini_file {
   my ($self, $filename, $defaults) = @_;
   my $inifile = undef;
   my $tree    = {};
   
+  ## Avoid deep-copying in NV divisions, to reduce size of packed files
+  ## See https://github.com/EnsemblGenomes/eg-web-common/commit/f702ab75235e66d7e9a979864b858fe0f88485f7
+  my $cow_from_defaults = $self->get_cow_defaults;
```

Comment:
should the function not be `_get_cow_defaults` you are missing an `_` at the beginning

Reply:
Thanks for spotting that!